FirebaseFunctions/Sources/Callable+Codable.swift

```diff
@@ -15,6 +15,12 @@
 import FirebaseSharedSwift
 import Foundation
 
+@available(macOS 12.0, iOS 15.0, tvOS 8.0,*)
```

Comment:
I think this availability attribute could match the other ones used (it's missing watchOS). 
```suggestion
@available(macOS 12.0, iOS 15.0, watchOS 8.0, tvOS 15.0, *)
```